validation/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,256 +1,38 @@
-= Validation Module - Mule 4
```

Comment:
Edit request 30 Nov 2020:

Type: Copyedit
Scope: Only validation-examples.adoc and index.adoc files (Green content)